sdk/resourcemanager/Azure.ResourceManager.Core/src/Generated/ManagementGroup/ManagementGroupOperations.cs

```diff
@@ -0,0 +1,339 @@
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        public virtual ManagementGroupsDeleteOperation StartDelete(string cacheControl = null, CancellationToken cancellationToken = default)
+        {
+            using var scope = _clientDiagnostics.CreateScope("ManagementGroupOperations.StartDelete");
+            scope.Start();
+            try
+            {
+                var originalResponse = _restClient.Delete(Id.Name, cacheControl, cancellationToken);
+                return new ManagementGroupsDeleteOperation(_clientDiagnostics, Pipeline, _restClient.CreateDeleteRequest(Id.Name, cacheControl).Request, originalResponse);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary>
+        /// Delete management group.
+        /// If a management group contains child resources, the request will fail.
+        /// .
+        /// </summary>
+        /// <param name="cacheControl"> Indicates whether the request should utilize any caches. Populate the header with &apos;no-cache&apos; value to bypass existing caches. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        public async virtual Task<ManagementGroupsDeleteOperation> StartDeleteAsync(string cacheControl = null, CancellationToken cancellationToken = default)
+        {
+            using var scope = _clientDiagnostics.CreateScope("ManagementGroupOperations.StartDelete");
+            scope.Start();
+            try
+            {
+                var originalResponse = await _restClient.DeleteAsync(Id.Name, cacheControl, cancellationToken).ConfigureAwait(false);
+                return new ManagementGroupsDeleteOperation(_clientDiagnostics, Pipeline, _restClient.CreateDeleteRequest(Id.Name, cacheControl).Request, originalResponse);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary>
+        /// List all entities that descend from a management group.
+        /// .
+        /// </summary>
+        /// <param name="skiptoken">
+        /// Page continuation token is only used if a previous operation returned a partial result.
+        /// If a previous response contains a nextLink element, the value of the nextLink element will include a token parameter that specifies a starting point to use for subsequent calls.
+        /// .
+        /// </param>
+        /// <param name="top"> Number of elements to return when retrieving results. Passing this in will override $skipToken. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        public virtual Pageable<DescendantInfo> GetDescendants(string skiptoken = null, int? top = null, CancellationToken cancellationToken = default)
```

Comment:
This feels like the method name should be ListDescendants

Reply:
Since this is in core, I think we can be opinionated about this, ratehr than necessarily deferring to the operationId